tools/crates/update.py

```diff
@@ -141,6 +142,82 @@ def matches_any_pattern(name):
                     print(f"Created placeholder for: {dir_path}")
 
 
+def filter_dependencies():
```

Comment:
this also seems unrelated to cargo bump?

Reply:
This is also related. Added info to description.